src/util/typeGuards.ts

```diff
@@ -2,22 +2,39 @@ import {
   WsFormattedMessage,
   WsMessage24hrMiniTickerRaw,
   WsMessage24hrTickerFormatted,
+  WsMessageAggTradeFormatted, WsMessageFuturesUserDataAccountConfigUpdateEventFormatted,
   WsMessageFuturesUserDataAccountConfigUpdateEventRaw,
+  WsMessageFuturesUserDataAccountUpdateFormatted,
   WsMessageFuturesUserDataAccountUpdateRaw,
   WsMessageFuturesUserDataEventFormatted,
-  WsMessageFuturesUserDataOrderTradeUpdateEventRaw,
+  WsMessageFuturesUserDataListenKeyExpiredFormatted,
+  WsMessageFuturesUserDataMarginCallFormatted,
+  WsMessageFuturesUserDataOrderTradeUpdateEventRaw, WsMessageFuturesUserDataTradeUpdateEventFormatted,
   WsMessageKlineFormatted,
   WsMessageKlineRaw,
   WsMessageMarkPriceUpdateEventFormatted,
+  WsMessageSpotBalanceUpdateFormatted,
+  WsMessageSpotOutboundAccountPositionFormatted,
   WsMessageSpotUserDataEventFormatted,
+  WsMessageSpotUserDataExecutionReportEventFormatted,
+  WsMessageSpotUserDataListStatusEventFormatted,
   WsMessageTradeFormatted,
   WsRawMessage,
   WsUserDataEvents,
 } from '..';
 
-/*
-  Use type guards to narrow down types with minimal efforts.
-*/
+/**
+ * Use type guards to narrow down types with minimal efforts.
+ *
+ * The file is organised by Typeguards starting with `WsFormattedMessage` typeguards in the first half
+ * and `WsRawMessage` typeguards in the second half.
```

Comment:
If there's a lot more type guards in future, we could even split this file into a folder, then separate files in that folder:
```
./typeGuards
./typeGuards/index.ts
./typeGuards/raw.ts
./typeGuards/formatted.ts
```

But probably overkill right now, more of a food for thought if this file grows a lot more.